notebooks/notebooks/manual_aoi/ch_zg_risch-rotkreuz.geojson

```diff
@@ -0,0 +1 @@
439173698425293,47.14117579260614],[8.438197374343872,47.14130715091039],[8.43801498413086,47.14138742526991],[8.436652421951294,47.14263531108927],[8.43600869178772,47.14251125357636]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[8.433841466903687,47.14829421511666],[8.433449864387512,47.14771047467649],[8.434587121009827,47.14730914940512],[8.435107469558716,47.14783816844558],[8.433841466903687,47.14829421511666]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[8.43314945697784,47.147393063121484],[8.431840538978573,47.145882595958284],[8.431325554847717,47.145203966302745],[8.430789113044735,47.14494856580146],[8.431089520454403,47.14468586686287],[8.431872725486752,47.14534990889538],[8.432484269142147,47.146108803918224],[8.433524966239926,47.14574395189415],[8.43391120433807,47.14620366503431],[8.433353304862973,47.14645905950539],[8.433803915977474,47.14717415749645],[8.43314945697784,47.147393063121484]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[8.43822956085205,47.15358407055327],[8.438546061515808,47.15349651861051],[8.438760638237,47.15376646997074],[8.438910841941833,47.15372269416764],[8.439554572105408,47.15439027124673],[8.439860343933105,47.154320960355435],[8.440235853195189,47.15435379184154],[8.440069556236267,47.15472952851569],[8.44024121761322,47.15479519096274],[8.44025731086731,47.155506528938616],[8.439699411392212,47.155232938536315],[8.43822956085205,47.15358407055327]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[8.455309867858887,47.14409478950448],[8.455438613891602,47.14393424884014],[8.455320596694945,47.14364965283454],[8.455621004104614,47.14302937421008],[8.455610275268553,47.14246017098687],[8.455867767333984,47.142146376860495],[8.456629514694214,47.14094956365465],[8.457080125808716,47.1410298385541],[8.456361293792725,47.14192744962894],[8.456811904907227,47.14216826953406],[8.456720709800718,47.14233976183178],[8.457552194595337,47.14237624948337],[8.457756042480469,47.14236530319053],[8.458008170127869,47.1423324642985],[8.458904027938843,47.14206610364695],[8.458941578865051,47.14213543052033],[8.458528518676756,47.14226678645357],[8.458308577537537,47.142887073975615],[8.457825779914856,47.142686393510445],[8.457369804382324,47.14245652222861],[8.45673680305481,47.14241638587113],[8.456082344055176,47.14235070812989],[8.456017971038818,47.144058303032445],[8.455309867858887,47.14409478950448]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[8.466988205909729,47.13418049411709],[8.466097712516785,47.13420603939551],[8.465250134468079,47.13463300866077],[8.464783430099487,47.134249831272776],[8.465631008148193,47.13382285893097],[8.465003371238708,47.13325720653239],[8.465341329574585,47.13300904745233],[8.466424942016602,47.133727975722806],[8.467084765434265,47.13383380698256],[8.466988205909729,47.13418049411709]]]}}]}
```

Comment:
why duplicate the files from `test-cases/manual_aoi/`?

Reply:
Opps, it was not intentional